pages/index.tsx

```diff
@@ -2,15 +2,32 @@ import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
 import Head from 'next/head'
 
+import React from "react";
+import RadioComponent from "./components/RadioComponent/RadioComponent";
+
+
 
 export default function Home() {
+
+
+
+  
+
   return (
     <>
       <Head>
         <title>Sign Up</title>
```

Comment:
It would match the link better if this was "Register" instead of "Sign Up"; I apologize for not mentioning that earlier.